build/DependencyVersions.props

```diff
@@ -17,6 +17,8 @@
     <PlatformAbstractionsVersion>2.0.0-preview2-002093</PlatformAbstractionsVersion>
     <DependencyModelVersion>2.0.0-preview2-002093</DependencyModelVersion>
     <CliCommandLineParserVersion>0.1.0-alpha-142</CliCommandLineParserVersion>
+    <CliProjectJsonMigration>1.1.0-alpha-2127</CliProjectJsonMigration>

```

Comment:
Not CliCommandLineParserVersion and ProjectJsonMigration, but ProjectJsonMigration and CliMigrateCommand

I can see the benefit for keep CLI and ProjectJsonMigration and CliMigrateCommand same version prefix-- you know it will work with certain version.

However, unlike SDK, this repo will not be in rapid development, it will not likely be consumed by other projects.

If we want to keep them in sync, every time we release, we will have to update all the versions of these libraries, on both sides. This discourages refactor out non CLI domain logic out of repo.

We also lost all the benefit from semantic versioning -- for CLI and SDK, there is marketing value out of it. But for a utility libraries, there is no

I agree and will fix the version 0 padding